guides/nestor/ISPsetup.md

```diff
@@ -0,0 +1,316 @@
+---
+Layout: Post
+Title: ISP setup ubuntu
+Author: Nestor N Torres
+---
+
+### Contact
+- Slack: @nanjuan on wcscusf.slack.com
+- [LinkedIn](https://www.linkedin.com/in/nestor-n-torres-737172a5/)
+- Email: nestor@nntorres.com
+
+### Table
+1. [Prerequisites](#preinfo)
+2. [Summary](#Summary)
+3. [VirtualBox VM Setup](#First)
+4. [Ubuntu Server installation](#Second)
+5. [Ubuntu Server Configuration](#Third)
+
+## Prerequisites <a id="preinfo"></a>
+1. If you have not setup your network adapter look here https://goo.gl/vs4cHC.
+2. Just copy the Prerequisites of peoples guides before yours here.
+3. Have your [virtual environment](link-to-guide) configured
+4. Have the [ISP](link-to-guide) gateway running.
+5. Have [pfSense](link-to-guide) running.
+
+## Summary <a id="Summary"></a>
+This tutorial will show you how to setup the ISP server you will need for your lab.
```

Comment:
This is fine but it would be awesome if it could be more verbose. Maybe move `supporting detail` into `summary` and get rid of `supporting detail`